kong/plugins/rate-limiting/handler.lua

```diff
@@ -67,6 +67,12 @@ local function get_identifier(conf)
 
   elseif conf.limit_by == "header" then
     identifier = kong.request.get_header(conf.header_name)
+
+  elseif conf.limit_by == "path" then
+    local req_path = kong.request.get_path()
+    if req_path == conf.path then
+      identifier = req_path
+    end
```

Comment:
Whether it is necessary to ignore case ?

Reply:
While some websites/services might decide to make their urls case-insensitive, the [html standard considers them sensitive](https://www.w3.org/TR/WD-html40-970708/htmlweb.html), so `/foo` and `/FOO`, in principle, point to different resources.

Case-sensitivity has a cluster of complexity in dealing with non-English letters - for example the Cyrillic and Greek alphabets have cased letters which can be used in paths, with encodings (see https://www.w3.org/International/articles/idn-and-iri/#path).